### DISCUSSION TITLE
Super-key no longer searches Firefox bookmarks?

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#377] [General]
Having gone back to SpiralLinux KDE version over the past week (deep down, always a KDE fan).
Was glad to notice bookmarks showing up in search results when hitting the "super-key".
Nowawadays, no more.
The only major thing i did, was upgrading system.
Why?

- - - - -

Btw, installation worked again via Rufus on usb.

Another positive point: 
Firefox on SpiralLinux doesn't see those stupid Ytube adblock warnings as it does on Windows.
Have a dozen extensions with uBlock Origin always!
No issues with Chrome based browsers with same extensions on either OSs.

**Comment by AlphaElwedritsch** [comment]
what do you mean with "super-key"?

**Reply by geckolinux** [reply]
On most hardware it would be the Windows key.

**Comment by geckolinux** [comment]
Hmm, did you check the detailed Plasma settings for file indexing and Krunner search results? Also check to make sure that the same Baloo packages that were originally installed are still installed. Was this an upgrade from a SpiralLinux 11.x installation to Debian 12, or did you start with SpiralLinux 12.x?

**Comment by dacomboman** [comment]
Yes, Windows key.

Had done a upgrade from existing default install.

But here now, from fresh install what i saw (likely before upgrading):

https://snipboard.io/wWpPJL.jpg

OS: Debian GNU/Linux bookworm 12.4 x86_64
 ,$$P'               `$$$.     Host: OptiPlex 7060
',$$P       ,ggs.     `$$b:    Kernel: 6.5.0-0.deb12.4-amd64
`d$$'     ,$P"'   .    $$$     Uptime: 2 hours, 15 mins
 $$P      d$'     ,    $$$P    Packages: 1913 (dpkg), 7 (flatpak)
 $$:      $.   -    ,d$$'      Shell: bash 5.2.15
 $$;      Y$b._   _,d$P'       Display (DELL P2222H): 1920x1080 @ 60Hz *
 Y$$.    `.`"Y$$$$P"'          Display (DELL P2222H): 1920x1080 @ 60Hz
 `$$b      "-.__               DE: KDE Plasma 5.27.5
  `Y$$                         WM: KWin (X11)
   `Y$$.                       WM Theme: Breeze
     `$$b.                     Theme: Breeze (Dark) [QT], Numix [GTK2/3]
       `Y$$b.                  Icons: breeze-dark [QT], breeze-dark [GTK2/3/4]
          `"Y$b._              Font: Ubuntu (11pt) [QT], Ubuntu (11pt) [GTK2/3/4]
             `"""              Cursor: breeze (24px)
                               Terminal: konsole 22.12.3
                               CPU: Intel(R) Core(TM) i5-8500 (6) @ 4.10 GHz
                               GPU: Intel UHD Graphics 630
                               Memory: 4.66 GiB / 7.55 GiB (62%)
                               Swap: 1.16 MiB / 7.18 GiB (0%)
                               Disk (/): 14.07 GiB / 86.48 GiB (16%) - btrfs
                               Local IP (eth0): 192.168.1.3/24 *
                               Locale: en_US.UTF-8

**Reply by dacomboman** [reply]
Perhaps it was the upgrade, an extra app. etc. who knows.

Did update via Discover this time instead of command:
sudo apt update && sudo apt upgrade -y

**Comment by dacomboman** [comment]
Think i found the problem.

Bookmarks no longer appear in search results via super-key after installing Vivaldi (Flathub) in Discover.
Issue remains even after setting Firefox back as default and a reboot.

**Reply by dacomboman** [reply]
Btw, did double-check to see if bookmarks plugin was still on.

**Reply by geckolinux** [reply]
> Bookmarks no longer appear in search results via super-key after installing Vivaldi (Flathub) in Discover.

Hi there, to me that doesn't sound like it could be the cause.

**Reply by dacomboman** [reply]
I might be wrong, but after removing SpiralLinux KDE and going over to the "dark side" so to speak, with KDE Neon's latest updated version. Even if disposition of search results is not the same, the outcome is.
And bookmarks fail to show in search results when Vivaldi is set as default browser (as opposed to default Firefox).
Now, this is probably not a Vivaldi issue, but any other Chrome based (imo).
Just haven't installed other browsers.

**Reply by geckolinux** [reply]
I'm not sure if this is the same issue?
https://forums.opensuse.org/t/bookmarks-entry-in-kde-main-starter-menu/135642/17

**Comment by dacomboman** [comment]
Could there still be a bug to iron out in latest Debian update?

**Reply by geckolinux** [reply]
It could very well be a Plasma / KDE bug, or it could just be an unexpected interaction between 3rd party components like the additional browser. Even if the bug is identified and fixed in KDE it almost surely won't get fixed in Debian, since they don't fix non-critical bugs during the course of the entire Debian Stable release lifecycle.

**Reply by dacomboman** [reply]
Mark that down as bug no.2 for me.
The other one being Dropbox not always loading at startup, even if checked in app. and system startup apps.

Hopefully Plasma 6 might improve on these issues.

**Reply by geckolinux** [reply]
If you're pretty sure that this is a reproducible bug then it's definitely worth reporting it over on https://discuss.kde.org and/or https://bugs.kde.org . I've found KDE to be a generally friendly and helpful community. Even if a fix doesn't arrive for Debian 12 there might be a workaround.

**Reply by dacomboman** [reply]
Will do!

Thank you for your coaching and ongoing assistance! ;)
